doc/user_guide/hive_user_guide.md

```diff
@@ -155,6 +198,18 @@ CREATE VIRTUAL SCHEMA <virtual schema name>
    SCHEMA_NAME     = '<schema name>';
 ```
 
+### Enabling Logging
+
+You can add `LogLevel` and `LogPath` parameters to the JDBC URL (the `TO` part of connection object) to enable Hive JDBC driver and connection logs.
+
+For example:
+
+```
+TO 'jdbc:hive2://<Hive host>:<port>;AuthMech=1;KrbAuthType=1;KrbRealm=EXAMPLE.COM;KrbHostFQDN=_HOST;KrbServiceName=hive;transportMode=http;httpPath=cliservice;LogLevel=6;LogPath=/tmp/'
+```
+
+This will create log files in the database `/tmp/` folder. Please check the `exasol_container_sockets` folder during the Virtual Schema creation or query run.
```

Comment:
Could you add a reference to a place where the meaning and available values for LogLevel are explained?